mini-project-1/products.json

```diff
@@ -2,20 +2,203 @@
   {
     "productId": 1,
     "productName": "Starbucks Espresso",
-    "price": 99,
+    "price": [59, 99, 129],
     "description": "Dark roasted beans for your delightful morning...",
-    "category": ["Dark roast", "SML"], //SML stands for available sizes
+    "category": ["Dark roast", "SML", "Starbucks"],
     "productImage": "assets/product_images/cafe1.png",
     "discount": "nodiscount",
+    "rating": "5"
   },
   {
     "productId": 2,
     "productName": "Starbucks Holiday",
-    "price": 99,
-    "description": "Holiday stuff, good stuff...",
-    "category": ["Light roast", "SL"],
+    "price": [59, 129],
```

Comment:
For the sake of project 1, I suggest removing the array of prices and use a single price instead.
Let's consider doing that later in React. But this is not even needed in the project description.
We need to filter by something, that's why we have our categories.